content/features/introductionToFeatures/chap2/sound.md

```diff
@@ -14,28 +14,37 @@ video-content:
 
 Adding sounds to your world is very easy.
 
-For continuous sounds we use
+First, we create an audio engine and unlock it:
 
 ```javascript
-const sound = new BABYLON.Sound("name", "url to sound file", scene, null, { loop: true, autoplay: true });
+BABYLON.CreateAudioEngineAsync().then((audioEngine) => {

```

Comment:
Should we use async/await instead ?

Reply:
Ya, it would probably be easier to tell users to put `async` in front of the playground's `createScene` function, right, instead of telling them to use `Promise.then(...)`.

Reply:
Done!